### PR TITLE
Add `/mv spawn tp` and `/mv spawn set`

### PR DESCRIPTION
**Pull request by zax71**
Renamed `/mvspawn` to `/mv spawn tp` and `/mvsetspawn` to `/mv spawn set`

### CODE REVIEW

File: src/main/java/org/mvplugins/multiverse/core/commands/SpawnCommand.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            @Flags("resolve=issuerAware")
+            @Syntax("[player]")
+            @Description("{@@mv-core.spawn.tp.player.description}")
+            Player player
+           ) {
+        // The player is in the world, so it must be loaded
+        LoadedMultiverseWorld world = worldManager.getLoadedWorld(player.getWorld().getName()).getOrNull();
+
+        // Teleport the player
+        safetyTeleporter.teleportSafely(issuer.getIssuer(), player, world.getSpawnLocation());
```
Comment (benwoo1110) [inline src/main/java/org/mvplugins/multiverse/core/commands/SpawnCommand.java:56]: need to check if teleport fails

File: src/main/java/org/mvplugins/multiverse/core/commands/SpawnCommand.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    void onSpawnTpCommand(
+            BukkitCommandIssuer issuer,
+
+            @Flags("resolve=issuerAware")
+            @Syntax("[player]")
+            @Description("{@@mv-core.spawn.tp.player.description}")
+            Player player
+           ) {
+        // The player is in the world, so it must be loaded
+        LoadedMultiverseWorld world = worldManager.getLoadedWorld(player.getWorld().getName()).getOrNull();
```
Comment (benwoo1110) [inline src/main/java/org/mvplugins/multiverse/core/commands/SpawnCommand.java:53]: this also, need to check if world is null

**Review by benwoo1110 — COMMENTED** [review]
i think dont need to change the command format. `/mv spawn` should be teleport to the world's spawn. Then i think `/mv setspawn` is fine? dont need to change to `/mv spawn set`

File: src/main/java/org/mvplugins/multiverse/core/commands/SpawnCommand.java (hunk 1 above)
Comment (zax71) [reply src/main/java/org/mvplugins/multiverse/core/commands/SpawnCommand.java:57]: How do I go about doing this? Using `.toAttempt().onFailiure()` causes the server to crash

**Review by benwoo1110 — APPROVED** [review]

**Pull request merged**